Skype/WebSDK/docs/PTAuthSignOut.md

```diff
@@ -0,0 +1,18 @@
+
+# Sign Out
```

Comment:
We should add something to this sample that explains that signInManager.state can be monitored to detect if the application gets disconnected - some guy already had that question.